core/services/ocr2/plugins/ccip/internal/cache/chain_health.go

```diff
@@ -25,99 +29,167 @@ import (
 // and is cached for a certain period of time based on defaultGlobalStatusDuration.
 // This may lead to some false-positives, but in this case we want to be extra cautious and avoid executing any reorged messages.
 //
-// Additionally, to reduce the number of calls to the RPC, we cache RMN curse state for a certain period of
-// time based on defaultRmnStatusDuration.
+// Additionally, to reduce the number of calls to the RPC, we refresh RMN state in the background based on defaultRMNStateRefreshInterval
 //
 //go:generate mockery --quiet --name ChainHealthcheck --filename chain_health_mock.go --case=underscore
 type ChainHealthcheck interface {
-	// IsHealthy checks if the chain is healthy and returns true if it is, false otherwise
-	// If forceRefresh is set to true, it will refresh the RMN curse state. Should be used in the Observation and ShouldTransmit phases of OCR2.
-	// Otherwise, it will use the cached value of the RMN curse state.
-	IsHealthy(ctx context.Context, forceRefresh bool) (bool, error)
+	job.ServiceCtx
+	// IsHealthy checks if the chain is healthy and returns true if it is, false otherwise.
+	IsHealthy(ctx context.Context) (bool, error)
 }
 
 const (
 	// RMN curse state is refreshed every 20 seconds or when ForceIsHealthy is called
-	defaultRmnStatusDuration    = 20 * time.Second
-	defaultGlobalStatusDuration = 30 * time.Minute
+	defaultRMNStateRefreshInterval = 10 * time.Second
+	defaultGlobalStatusDuration    = 30 * time.Minute
```

Comment:
Naming nit (feel free to ignore):

```suggestion
	defaultGlobalStatusExpirationDuration    = 30 * time.Minute
```